license_test.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	pkggodevclient "github.com/guseggert/pkggodev-client"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/modfile"
+)
+
+var (
+	acceptedLicenses = map[string]struct{}{
+		"MIT":          {},
+		"Apache-2.0":   {},
+		"BSD-3-Clause": {},
+		"BSD-2-Clause": {},
+		"ISC":          {},
+		"CC-BY-SA-4.0": {},
+		"MPL-2.0":      {},
+	}
+
+	knownUndectedLicenses = map[string]string{}
+)
+
+func TestLicenses(t *testing.T) {
```

Comment:
i think there is an official github action that we can use to check licenses as a workflow https://github.com/actions/dependency-review-action

Reply:
Sounds good to me to use the action instead. Less code to maintain.